pkgs/by-name/bl/blockbench/package.nix

```diff
@@ -0,0 +1,83 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, imagemagick
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, electron # Note: running with electron_26 leads to a crash after a few seconds
```

Comment:
This comment is outdated. Does `electron_28` work fine?

Reply:
I am not sure I get what you mean, does it not crash for you with `electron_26`? And yes, it seems to work with `electron_28`.

Reply:
Since `electron_28` is the default I don't think the comment is needed.

Reply:
Ah, I see you clarified the comment. Sounds good to me now.